### DISCUSSION TITLE
Linux install instructions - fedora

### DISCUSSION THREAD

**Discussion by jpmorr** [quarto-dev/quarto-cli#3043] [Q&A] [answered]
Hi,

I'm just wondering what the instructions for installation on Linux, specifically Fedora are. I've downloaded the latest tarball from the website and extracted it's contents to `~/quarto`. I didn't see anything on the site regarding installation.

I've installed the quarto VS Code extension and tries to verify the installation but nothing happens. I assume it's checking a specific path and that I've extracted quarto-cli to the wrong directory. I didn't see and setup or config file that I should run in the extracted folder.

Any help would be appreciated.

**Comment by dragonstyle** [comment, marked as answer]
Hi,

Sorry we don't have better instructions for this yet. I just put together some instructions for installation of the Debian based Linux tarball which I believe should mostly be accurate for Fedora (definitely the concepts of each step should be). I'd very much like to put together a similar set of instructions for the Fedora version - if you find that these are successful or you have suggestions / improvements specifically for Fedora, I would really appreciate it (I don't yet have a Fedora system to test on so I didn't write the docs for that just yet)...

https://quarto.org/docs/download/tarball.html?version=1.2.242&idPrefix=download-pre

Charles

**Reply by jpmorr** [reply]
@dragonstyle Thanks for this, really helpful. I mostly followed the instructions, but changed the paths you had used as they were all in the home directory, so I'm using `/bin/...` and `/opt` on my machine as these are already on my path. 

This mostly worked and I was able to run quarto check. But...
```
[✓] Checking Quarto installation......OK
      Version: 1.2.242
      Path: /opt/quarto-1.2.242/bin

tcc: error: mprotect failed: did you mean to configure --with-selinux?
```
I'm not sure about the error and whether it relates to this being outside the home directory?

**Reply by cscheid** [reply]
<strike>The issue is that @dragonstyle linked to a slightly older version of the release candidate. We just fixed an issue (actually a bug on upstream deno and Fedora). Can you try the [very latest v1.2](https://github.com/quarto-dev/quarto-cli/releases/tag/v1.1.251)?</strike>

EDIT: I'm wrong. There's something funky happening with our auto-release building. Let me fix that and I'll get back to you.

**Reply by cscheid** [reply]
There should be a new quarto release (1.2.243) coming up in half an hour or so.

**Reply by jpmorr** [reply]
@dragonstyle @cscheid, just tried 1.2.245 and that seems to be working fine:
```
$ quarto check

[✓] Checking Quarto installation......OK
      Version: 1.2.245
      Path: /opt/quarto-1.2.245/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.9.13 (Conda)
      Path: /home/jmorrise/anaconda3/bin/python
      Jupyter: 4.11.1
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

My RStudio install seems to have stopped working for some other unrelated reason which is why it's not getting found.

Thanks!

**Comment by romunov** [comment]
I mapped `quarto` to `~/.local/bin/quarto` to expose it system wide.

**Comment by statick88** [comment]
Strange, I could not follow all the steps, I have an error in this section:

```
~/bin/quarto check

[✓] Checking Quarto installation......OK
      Version: 1.2.242
      Path: /home/statick/opt/quarto-1.2.242/bin

tcc: error: mprotect failed: did you mean to configure --with-selinux?

```

**Reply by mcanouil** [reply]
The method is one year old, there is no guarantee your system is the same a the one used at that time.
Also, you are not even using the version which worked as mentioned in <https://github.com/quarto-dev/quarto-cli/discussions/3043#discussioncomment-3971859>, i.e., 1.2.245.

Note that current stable version is 1.3.450.

**Reply by statick88** [reply]
I'm create a gist with this solution to my problem

[Instalación de Quarto en Linux desde un archivo tarball ](https://gist.github.com/statick88/f747ad29983944266c500a441be71cf4#file-instalacion-de-quarto-en-linux-desde-un-archivo-tarball)
